BUG: Bug with `pip pre` / sklearn

@agramfort maybe a quick one for you:
```
____________________ test_get_coef_multiclass_full[4-10-2] ____________________
mne\decoding\tests\test_base.py:302: in test_get_coef_multiclass_full
    scores = cross_val_multiscore(time_gen, X, y, cv=cv, verbose=True)
mne\decoding\base.py:441: in cross_val_multiscore
    scores = parallel(p_func(clone(estimator), X, y, scorer, train, test,
mne\decoding\base.py:441: in <genexpr>
    scores = parallel(p_func(clone(estimator), X, y, scorer, train, test,
mne\decoding\base.py:474: in _fit_and_score
    X_train, y_train = _safe_split(estimator, X, y, train)
c:\hostedtoolcache\windows\python\3.8.6\x64\lib\site-packages\sklearn\utils\metaestimators.py:198: in _safe_split
    if _is_pairwise(estimator):
c:\hostedtoolcache\windows\python\3.8.6\x64\lib\site-packages\sklearn\base.py:844: in _is_pairwise
    pairwise_tag = _safe_tags(estimator, key="pairwise")
c:\hostedtoolcache\windows\python\3.8.6\x64\lib\site-packages\sklearn\utils\_tags.py:62: in _safe_tags
    raise ValueError(
E   ValueError: The key pairwise is not defined in _get_tags() for the class GeneralizingEstimator.
```

--- a/mne/fixes.py
+++ b/mne/fixes.py
@@ -278,6 +278,28 @@ def is_regressor(estimator):
     return getattr(estimator, "_estimator_type", None) == "regressor"
 
 
+_DEFAULT_TAGS = {
+    'non_deterministic': False,
+    'requires_positive_X': False,
+    'requires_positive_y': False,
+    'X_types': ['2darray'],
+    'poor_score': False,
+    'no_validation': False,
+    'multioutput': False,
+    "allow_nan": False,
+    'stateless': False,
+    'multilabel': False,
+    '_skip_test': False,
+    '_xfail_checks': False,
+    'multioutput_only': False,
+    'binary_only': False,
+    'requires_fit': True,
+    'preserves_dtype': [np.float64],
+    'requires_y': False,
+    'pairwise': False,
+}
+
+
 class BaseEstimator(object):
     """Base class for all estimators in scikit-learn
 
@@ -397,6 +419,9 @@ class BaseEstimator(object):
     # __getstate__ and __setstate__ are omitted because they only contain
     # conditionals that are not satisfied by our objects (e.g.,
     # ``if type(self).__module__.startswith('sklearn.')``.
+
+    def _more_tags(self):
+        return _DEFAULT_TAGS
 
     def _get_tags(self):
         collected_tags = {}

